core/indexer/processTransactions_test.go

```diff
@@ -343,50 +339,99 @@ func TestAlteredAddresses(t *testing.T) {
 	// addresses marked with a comment should be added to the altered addresses map
 
 	// normal txs
-	addressSh0NormalTx1 := []byte("sender in shard 0 - tx 1") // should be added
-	addressSh1NormalTx1 := []byte("receiver in shard 1 - tx 1")
-	expectedAlteredAccounts[hex.EncodeToString(addressSh0NormalTx1)] = struct{}{}
-	tx1 := &transaction.Transaction{SndAddr: addressSh0NormalTx1, RcvAddr: addressSh1NormalTx1}
+	address1 := []byte("address1") // should be added
+	address2 := []byte("address2")
+	expectedAlteredAccounts[hex.EncodeToString(address1)] = struct{}{}
+	tx1 := &transaction.Transaction{
+		SndAddr: address1,
+		RcvAddr: address2,
+	}
 	tx1Hash := []byte("tx1Hash")
 
-	addressSh0NormalTx2 := []byte("receiver in shard 0 - tx 2") // should be added
-	addressSh1NormalTx2 := []byte("sender in shard 1 - tx 2")
-	expectedAlteredAccounts[hex.EncodeToString(addressSh0NormalTx2)] = struct{}{}
-	tx2 := &transaction.Transaction{SndAddr: addressSh1NormalTx2, RcvAddr: addressSh0NormalTx2}
+	address3 := []byte("address3") // should be added
+	address4 := []byte("address4")
+	expectedAlteredAccounts[hex.EncodeToString(address4)] = struct{}{}
```

Comment:
the comment is added to `address3` but you add `address4`

Reply:
done